scipy/stats/tests/test_multicomp.py

```diff
@@ -0,0 +1,174 @@
+import copy
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from scipy import stats
+from scipy.stats._multicomp import pvalue_dunnett, DunnettResult
+
+
+class TestDunnett:
+
+    @pytest.mark.parametrize(
+        'rho, n_groups, df, statistic, pvalue, alternative',
+        [
+            # From Dunnett1995
+            # Tables 1a and 1b pages 1117-1118
+            (0.5, 1, 10, 1.81, 0.05, "greater"),  # different than two-sided
+            (0.5, 3, 10, 2.34, 0.05, "greater"),
+            (0.5, 2, 30, 1.99, 0.05, "greater"),
+            (0.5, 5, 30, 2.33, 0.05, "greater"),
+            (0.5, 4, 12, 3.32, 0.01, "greater"),
+            (0.5, 7, 12, 3.56, 0.01, "greater"),
+            (0.5, 2, 60, 2.64, 0.01, "greater"),
+            (0.5, 4, 60, 2.87, 0.01, "greater"),
+            (0.5, 4, 60, [2.87, 2.21], [0.01, 0.05], "greater"),
+            # Tables 2a and 2b pages 1119-1120
+            (0.5, 1, 10, 2.23, 0.05, "two-sided"),  # two-sided
+            (0.5, 3, 10, 2.81, 0.05, "two-sided"),
+            (0.5, 2, 30, 2.32, 0.05, "two-sided"),
+            (0.5, 3, 20, 2.57, 0.05, "two-sided"),
+            (0.5, 4, 12, 3.76, 0.01, "two-sided"),
+            (0.5, 7, 12, 4.08, 0.01, "two-sided"),
+            (0.5, 2, 60, 2.90, 0.01, "two-sided"),
+            (0.5, 4, 60, 3.14, 0.01, "two-sided"),
+            (0.5, 4, 60, [3.14, 2.55], [0.01, 0.05], "two-sided"),
+            # From Kwong2000 Table 2
+            (0.5, 9, 30, 2.856, 0.05, "two-sided"),
+            (0.5, 17, 20, 3.162, 0.05, "two-sided"),
+            # balanced designs
+            (0.3, 10, 10, 3.401, 0.05, "two-sided"),
+            (0.3, 13, 20, 3.168, 0.05, "two-sided"),
+            (0.1, 12, 20, 3.184, 0.05, "two-sided"),
+            (0.1, 15, 30, 3.157, 0.05, "two-sided"),
```

Comment:
These values appear to be for a step-up version of the test. Would we expect them to be different from the critical values of the original single-step test (which is what we implemented)?

Reply:
All cases being equal, I did not really pay attention.

Reading at the introduction of Kwong2000. Isn't the stepping up-down a way to get to the empirical value of the multivariate t-distribution? Meaning it would translate in a "resolution" problem to compare to values from such tables??

Reply:
It's a different computation with the multivariate t distribution than the one we use in the single-step Dunnett's test that we've implemented. They aren't necessarily numerically _far_ from each other, but they aren't strictly useful points of reference.

Reply:
Ok thanks 👍 now the follow up question is: what do we do here? Is it ok if we assume it's an implementation detail and roll with this implementation/tests?

Reply:
Delete the rows from Kwong2000. They are irrelevant to what we've implemented.

Reply:
```suggestion
```

Reply:
Yes, and then probably also delete the Kwong2000 citation in the `References` because it's not referenced for anything but this.

Reply:
Thanks. This is what I had in mind, but I only had a few minutes to look today. I think I can look more carefully Friday, when I'll try to write some tests for the general case.

Reply:
The Kwong citation was not deleted, but I have suggested its removal in another comment, so resolving.